### PR TITLE
DR-1605: Update helm charts on jade-data-repo version bump

### PR DESCRIPTION
**Pull request by snf2ye**
### Update hem charts on jade-data-repo version bump
- datarepo-helm PR - https://github.com/broadinstitute/datarepo-helm/pull/93
- example passing action: https://github.com/DataBiosphere/jade-data-repo/actions/runs/575490207

- Updated Slack notifications in jade-spam channel:
image

### CODE REVIEW

File: .github/workflows/helmtagbumper.yaml [hunk 1]
```diff
@@ -61,14 +61,80 @@ jobs:
...
         if: always()
         uses: broadinstitute/action-slack@v3.8.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
         with:
           status: ${{ job.status }}
-          author_name: api_helm_bumper
-          fields: repo,message,commit,author,action,eventName,ref,workflow,job,took
+          fields: job,repo,message,author,tfook
```
Comment (samanehsan) [inline .github/workflows/helmtagbumper.yaml]: Looks like there's a small typo in `took`

File: .github/workflows/helmtagbumper.yaml [hunk 2]
```diff
@@ -61,14 +61,80 @@ jobs:
...
+          author_name: "[API] [datarepo-helm-definitions] Verion update for Integration namespaces"
```
Comment (samanehsan) [inline .github/workflows/helmtagbumper.yaml]: Also another small typo in "version" (here and in the author_name below)

**Review by samanehsan — APPROVED** [review]
Just had a couple of minor comments, otherwise this looks great! :octocat:

File: .github/workflows/helmtagbumper.yaml (hunk 1 above)
Comment (snf2ye) [reply .github/workflows/helmtagbumper.yaml]: Thank you for catching this!

File: .github/workflows/helmtagbumper.yaml (hunk 2 above)
Comment (snf2ye) [reply .github/workflows/helmtagbumper.yaml]: Thanks!

File: .github/workflows/helmtagbumper.yaml [hunk 3]
```diff
@@ -61,14 +61,80 @@ jobs:
...
+        if: always()
+        uses: broadinstitute/action-slack@v3.8.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+        with:
+          status: ${{ job.status }}
+          fields: job,repo,message,author,took
+          author_name: "[API] [datarepo-helm] Version update for Helm Charts"
+          only_mention_fail: smark,fb,muscles
```
Comment (nmalfroy) [inline .github/workflows/helmtagbumper.yaml:140]: lol...leaving yourself off I see :)

**Review by nmalfroy — APPROVED** [review]
lgtm!

File: .github/workflows/helmtagbumper.yaml (hunk 3 above)
Comment (snf2ye) [reply .github/workflows/helmtagbumper.yaml:140]: just copying the other alerts 😂

**Pull request merged**